### DISCUSSION TITLE
Bug Report, Fail exhibitionism mission for picking up suit but not wearing one.

### DISCUSSION THREAD

**Discussion by McFacePunch** [FakeFishGames/Barotrauma#13887] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

While doing the exhibitionist mission, i moved a suit by picking it up with `E` and it resulted in an instant mission failed. This is not expected behavior because it was never worn, only picked up, making the mission criteria more strict than the mission implies.

### Reproduction steps

Get mission for exhibitionism, remove all clothing, pick up a suit with E but do not wear it.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! [Created a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13897), we'll look into it!

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, coming up in the next unstable build!
